lib/Service/ParticipantService.php

```diff
@@ -1031,10 +1031,14 @@ public function endCallForEveryone(Room $room, Participant $moderator): void {
 		$this->dispatcher->dispatch(Room::EVENT_AFTER_END_CALL_FOR_EVERYONE, $event);
 	}
 
-	public function changeInCall(Room $room, Participant $participant, int $flags, bool $endCallForEveryone = false, bool $silent = false): void {
+	public function changeInCall(Room $room, Participant $participant, int $flags, bool $endCallForEveryone = false, bool $silent = false): bool {
+		if ($room->getType() === Room::TYPE_CHANGELOG || $room->getType() === Room::TYPE_NOTE_TO_SELF) {
```

Comment:
Would it make sense to check for `ONE_TO_ONE_FORMER` here as well?

Reply:
We could of course, on the other hand we are not blocking the calling there yet. So let's make that a new issue?

Reply:
https://github.com/nextcloud/spreed/issues/10595